web_src/css/modules/toast.css

```
@@ -29,12 +29,23 @@
   background: transparent;
   border: none;
   display: flex;
-  width: 30px;
-  height: 30px;
+  min-width: 30px;
+  min-height: 30px;
   justify-content: center;
   align-items: center;
 }
 
+.toast-duplicate-number::before {
+  content: "(";
+}
+.toast-duplicate-number {
+  display: inline-flex;
+  margin: 0 2px;
```

```suggestion
  margin: 0 2px;
  user-select: none;
```

Given that the braces are not selectable because they are pseudos, this likely should not as well.